torch/ao/quantization/pt2e/_numeric_debugger.py

```diff
@@ -318,10 +318,28 @@ def compare_results(
             )
             continue
         actual_name, actual_stack, actual_stats = actual_results[debug_handle]
+        for a, b in zip(actual_stats, ref_stats):
+            if (
+                isinstance(a, torch.Tensor)
+                and isinstance(b, torch.Tensor)
+                and a.shape != b.shape
+            ):
+                log.warning(
+                    "Cannot compare tensors with different shapes: actual=%s vs ref=%s",
+                    a.shape,
+                    b.shape,
+                )
```

Comment:
should we raise exception here instead of ignore the error

Reply:
I don't think so, because this type of error would typically indicate the debug handle was placed on the wrong nodes to compare before and after. Debug handles should always produce tensors of the same shape.

But debug handle errors should not prevent reporting of other accuracy errors.